config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root 'users#index'
+  resources :users, only: %i[show index]
+  get '/users/:user_id/posts', to: 'posts#index', as: 'user_posts'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'user_post'
```

Comment:
**OPTIONAL** - Good job done so far, but I suggest you use the `resources` keyword as you did with the User route, its a cleaner way to generate your routes. Please fix this 😄 
```ruby
      resources :users, only: %i[index show] do
           resources :posts, only: %i[index show]
end

Reply:
Alright